cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-westend/src/tests/mod.rs

```diff
@@ -13,6 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+mod claim_assets;
```

Comment:
@programskillforverification can you please add this test for other runtime tests under `cumulus/parachains/integration-tests/emulated/tests/` directory?

Reply:
@bkontur Yes. I just want to run the check and find issues. I will do other parts right now.